exp-player/addon/components/exp-player.js

```diff
@@ -142,6 +143,7 @@ able to continue the study.
         next() {
             console.log('next');
             var frameIndex = this.get('frameIndex');
+            this.sendAction('updateFrameIndex', frameIndex);
```

Comment:
Does this create any redundancy with L150 (will frameIndex get incremented twice?)?